play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
 The boundaries of speed and maneuvering were meant to be broken - by you.
-Passives: Gain Piloting skill at 10.
-At level 6, the Feat ". . . Who Needs Nano-Technologies" is automatically 
-unlocked.
-Weapon Proficiencies: Pistols, Shotguns, SMGs
-Armour Proficiencies: Concealed, Recon, Light, Medium.
-Starting Wealth: $1700+100*Luck -1050
+Class Primary Attribute: Dexterity
+Passives: Gain Skill [Piloting] at 10.
+At level 6, the Feat ". . . Who Needs Nano-Technologies" is automatically unlocked.
+Weapon Proficiencies: Pistols, Submachine Guns, Shotguns.
+Armor Proficiencies: Concealed, Recon, Light, Medium.
+Starting Wealth: $850 + 100*Luck + 1820
 Class Feats:
-----------------------
-Planes, Trains, Automobiles -
-This is sorta familiar to something you once saw, or rode on. Maybe? 
-Unfamiliar vehicles can be operated [albeit pretty poorly] after you pass a DC
-50 check, or take up to 3 actions examining.
-|
-|
-|
-V
-You have flown, operated and driven just about everything there is. Except 
-that. Make a DC 40 check, or take up to 2 actions examining.
-----------------------
-Extensive training <Vehicle> -
-Your complete knowledge of <vehicle> makes your operation of it seem easy. Add
-25 to any check concerning your operation, knowldege, or maintenence of the 
-vehicle.
-|
-|
-|
-V
-You are THE guru accross the holonets for <vehicle>. Add 50 onto a pertinent 
-roll about the vehicle.
-
-"Adept" - Your specialized skills with technology are combined with battle 
+	----------------------
+	Planes, Trains, Automobiles -
+	This is sorta familiar to something you once saw, or rode on. Maybe? 
+	Unfamiliar vehicles can be operated [albeit pretty poorly] after you pass 
+	a DC 50 Luck check, or take up to 3 actions examining.
+	|
+	|
+	V
+	You have flown, operated and driven just about everything there is. Except 
+	that. Make a DC 50 Piloting OR Luck check, or take up to 2 actions examining.
+	----------------------
+	Extensive training <Vehicle> -
+	Your complete knowledge of <Vehicle> makes your operation of it seem easy. 
+	Add	25 to any check concerning your operation, knowledge, or maintenance of the 
+	vehicle.
+	|
+	|
+	V
+	You are THE guru across the holonets for <Vehicle>. Add 50 onto a pertinent 
+	roll about the vehicle.
+======================================
+
+
+=====   ADEPT   =====
+Your specialized skills with technology are combined with battle 
 training, providing you an edge in combat situations.
-Passives: Gain 100 nanites. any number of these can be put into a permanent 
-passive CS Class B unit that is implanted in your body. When not wearing armor,
-it covers 75% excluding head, hands, and feet. when wearing armor, it extends 
-your DR by 10 and you gain 5% coverage.
-Weapon Proficiencies: Pistols, SMGs, Carbines and Battle Rifles.
-Armour Proficiencies: Concealed, Recon, Light, Medium.
-Starting Wealth: $1700+100*Luck - 400
+Class Primary Attribute: Strength
```

Comment:
I'm really not sure about Adepts, but see the Str vs Fort argument I made for Soldier.